rq/worker.py

```diff
@@ -160,11 +161,22 @@ def __init__(self, queues, name=None,
 
     def validate_queues(self):
         """Sanity check for the given queues."""
-        if not iterable(self.queues):
-            raise ValueError('Argument queues not iterable.')
         for queue in self.queues:
             if not isinstance(queue, self.queue_class):
-                raise NoQueueError('Give each worker at least one Queue.')
+                raise NoQueueError('Give each worker at least one Queue.\
+                        {} was passed'.format(queue))
+
+    def process_queue_args(self, queue_args):
+        """ allow for a variety arguements to queues """
+        if isinstance(queue_args, six.text_type):
```

Comment:
Can you please change this line to use `rq.compat.text_type` so we don't need need to depend on `six`?